pkg/sentry/platform/ring0/lib_arm64.go

```diff
@@ -16,10 +16,27 @@
 
 package ring0
 
-// LoadFloatingPoint loads floating point state by the most efficient mechanism
-// available (set by Init).
-var LoadFloatingPoint func(*byte)
+// CPACR_EL1, Architectural Feature Access Control Register.
+// Get the value of CPACR_EL1.
+func GetCpacrEl1() (value uintptr)
```

Comment:
This should be named CPACREL1. Not need for 'Get'.

FWIW, with the proper capitalization, this is admittedly not super readable. It looks like there is no CPACR_EL2. Would it make sense to just name this CPACR?

https://golang.org/doc/effective_go.html#Getters

Reply:
Done.
CPACR is a Arm32 register. On Aarch64, the name is cpacr_el1.
Please check it: https://developer.arm.com/docs/ddi0595/b/aarch32-system-registers/cpacr